tests/specs/functional/lemmas-spec.k

```diff
@@ -57,4 +57,16 @@ module LEMMAS-SPEC
     rule <k> runLemma ( #range(M [ 32 := (0 : 1 : 2 : .ByteArray)         ] , 33 , 2) ) => doneLemma( (1 : 2 : .ByteArray)                      ) ... </k>
     rule <k> runLemma ( #range(M [ 32 := (0 : 1 : 2 : .ByteArray)         ] , 33 , 3) ) => doneLemma( (1 : 2 :  .ByteArray) ++ #range(M, 35, 1) ) ... </k>
 
+    //bool2Word simplification
+    rule <k> runLemma ( bool2Word( B:Bool ) ==Int 1 ) => doneLemma( B ==K true ) ... </k>
+    rule <k> runLemma ( 1 ==Int bool2Word( B:Bool ) ) => doneLemma( B ==K true ) ... </k>
+
+    //#buf simplification
+    rule <k> runLemma ( #buf ( 32 , bool2Word ( B:Bool ) ) ==K #buf(32, 1) )
+             => doneLemma( B ==K true ) ... </k>
+    rule <k> runLemma ( #buf ( 32 , I:Int ) ==K #buf(32, 1) )
+             => doneLemma( I ==K 1 ) ... </k>
+    rule <k> runLemma ( #buf(32, 1) ==K #buf ( 32 , I:Int ))
+             => doneLemma( I ==K 1 ) ... </k>
```

Comment:
These can all be one line, and lined up horizontally, formatting.

Makes scanning the file a lot easier.